jinahub/indexers/searcher/FaissSearcher/README.md

```diff
@@ -1,25 +1,31 @@
 # FaissSearcher
 
-**FaissSearcher** is a Faiss-powered vector Searcher.
-
-Faiss is a library for efficient similarity search and clustering of dense vectors. It contains algorithms that search in sets of vectors of any size, up to ones that possibly do not fit in RAM. It also contains supporting code for evaluation and parameter tuning. Faiss is written in C++ with complete wrappers for Python/numpy. Some of the most useful algorithms are implemented on the GPU. It is developed by Facebook AI Research.
-
-
+**FaissSearcher** wraps Faiss-powered vector Searcher.
 
+**FaissSearch** is defined as a vector searcher,
+These usually implement a form of similarity search,
+based on the embeddings created by the encoders you have chosen in your Flow.
+Vector searcher is meant to be used together with a **Storage**.
+To understand Jina's storage-search workflow,
+please read the documentation [here](https://docs.jina.ai/advanced/experimental/indexers/).
 
+[Faiss](https://github.com/facebookresearch/faiss) is a library for efficient similarity search and clustering of dense vectors.
+It contains algorithms that search in sets of vectors of any size, up to ones that possibly do not fit in RAM.
+It also contains supporting code for evaluation and parameter tuning.
+Faiss is written in C++ with complete wrappers for Python/numpy.
+Some of the most useful algorithms are implemented on the GPU.
+It is developed by Facebook AI Research.
 
 
 ## Usage
 
-Check [tests](tests) for an example on how to use it.
-
 ### Loading data
 
-Since this is a "Searcher"-type Executor, it does not _index_ new data. Rather they are write-once classes, which take as data source a `dump_path`. 
-
-This can be provided in different ways:
+Since this is a "Searcher"-type Executor, it does not _index_ new data.
+Rather they are write-once classes, which take as data source a `dump_path`. 
+Then we can perform search operations on the loaded data.
 
-- in the YAML definition
+#### Loading data with YAML
   
 ```yaml
 jtype: FaissSearcher
```

Comment:
This part is not very clear. Even after setting the YAML, how would they use the YAML files in their codes?
I would suggest to either removing this part because the usage is almost the same as the next section. The user should learn how to use YAML to config their executors from docs.jina.ai